:farmer: Hci/Kci/Rci release/debug/connext ros2param test verb load failing

### Generated by Generative AI

_No response_

### Operating System:

Ubuntu 24.04 / 22.04

### ROS version or commit hash:

Rolling, Kilted, Humble

### RMW implementation (if applicable):

_No response_

### RMW Configuration (if applicable):

_No response_

### Client library (if applicable):

_No response_

### 'ros2 doctor --report' output

_No response_

### Steps to reproduce issue

1. Go to any of the jobs marked down, for example https://build.ros2.org/job/Kci__nightly-release_ubuntu_noble_amd64/75
2. Build the job
3. See tests failing

### Expected behavior

To run tests without failing

### Actual behavior

Finished with `ros2param.ros2param.test.test_verb_load.test_verb_load` failing

### Additional information

### Reference Builds
- https://build.ros2.org/job/Kci__nightly-release_ubuntu_noble_amd64/75
- https://build.ros2.org/job/Kci__nightly-debug_ubuntu_noble_amd64/75
- https://build.ros2.org/job/Kci__nightly-connext_ubuntu_noble_amd64/74
- https://build.ros2.org/job/Rci__nightly-release_ubuntu_noble_amd64/614
- https://build.ros2.org/job/Rci__nightly-debug_ubuntu_noble_amd64/597
- https://build.ros2.org/job/Rci__nightly-connext_ubuntu_noble_amd64/618
- https://build.ros2.org/job/Hci__nightly-connext_ubuntu_jammy_amd64/419/

### Test Failing
- [ros2param.ros2param.test.test_verb_load.test_verb_load](https://build.ros2.org/view/Rci/job/Rci__nightly-release_ubuntu_noble_amd64/598/testReport/junit/ros2param.ros2param.test/test_verb_load/test_verb_load/)

### Log Output
```
========================================================================================================
FAIL: test_verb_load.TestVerbLoad.test_verb_load_wildcard[rmw_connextdds]
--------------------------------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/ws/src/ros2/ros2cli/ros2param/test/test_verb_load.py", line 363, in test_verb_load_wildcard
    loaded_params = yaml.safe_load(param_dump_command.output)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 125, in safe_load
    return load(stream, SafeLoader)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 81, in load
    return loader.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/composer.py", line 58, in compose_document
    self.get_event()
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 118, in get_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/parser.py", line 193, in parse_document_end
    token = self.peek_token()
            ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/scanner.py", line 129, in peek_token
    self.fetch_more_tokens()
  File "/usr/lib/python3/dist-packages/yaml/scanner.py", line 223, in fetch_more_tokens
    return self.fetch_value()
           ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/yaml/scanner.py", line 577, in fetch_value
    raise ScannerError(None, None,
yaml.scanner.ScannerError: mapping values are not allowed here
  in "<unicode string>", line 2, column 15:
    /foo/test_node:
                  ^

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/ws/src/ros2/ros2cli/ros2param/test/test_verb_load.py", line 368, in test_verb_load_wildcard
    self.fail(f'Failed to parse YAML output: {e}')
AssertionError: Failed to parse YAML output: mapping values are not allowed here
  in "<unicode string>", line 2, column 15:
    /foo/test_node:
                  ^
```

### Flakyness last 15 days
job_name|last_fail|first_fail|build_count|failure_count|failure_percentage
|--|--|--|--|--|--|
Kci__nightly-release_ubuntu_noble_amd64|2025-12-16|2025-12-04|5|5|100.0
Kci__nightly-debug_ubuntu_noble_amd64|2025-12-16|2025-12-04|5|5|100.0
Kci__nightly-connext_ubuntu_noble_amd64|2025-12-16|2025-12-04|4|4|100.0
Rci__nightly-release_ubuntu_noble_amd64|2025-12-08|2025-12-03|6|6|100.0
Rci__nightly-debug_ubuntu_noble_amd64|2025-12-08|2025-12-03|6|6|100.0
Rci__nightly-connext_ubuntu_noble_amd64|2025-12-08|2025-12-03|6|6|100.0
Hci__nightly-connext_ubuntu_jammy_amd64|2025-12-06|2025-12-06|6|1|16.67

Doing an investigation I found that in humble the problem was solved between job [#419](https://build.ros2.org/job/Hci__nightly-connext_ubuntu_jammy_amd64/419) and [#420](https://build.ros2.org/job/Hci__nightly-connext_ubuntu_jammy_amd64/420)
repos differences:
```diff
@@ -239,7 +239,7 @@ repositories:
   ros2/message_filters:
     type: git
     url: https://github.com/ros2/message_filters.git
-    version: aa8b926552eb37d7c3b98a0adec5f70a6e9163e3
+    version: 1cfb762aba2e6f67ff23b4063ce57b39b26af328
   ros2/mimick_vendor:
     type: git
     url: https://github.com/ros2/mimick_vendor.git
@@ -275,7 +275,7 @@ repositories:
   ros2/rclcpp:
     type: git
     url: https://github.com/ros2/rclcpp.git
-    version: b6238def5824b3bdc846ce96ec3eb6526e0e81ec
+    version: 2ba0b01d09e565f94c3ffcb02879fb4f5f6ebbc8
   ros2/rclpy:
     type: git
     url: https://github.com/ros2/rclpy.git
@@ -339,7 +339,7 @@ repositories:
   ros2/rosidl:
     type: git
     url: https://github.com/ros2/rosidl.git
-    version: 0eebc5df2e9647887f557f22c7613db5e354335d
+    version: 9e247ad4fb2510839c4eb3dc0f18e47f1d2016a0
   ros2/rosidl_dds:
     type: git
     url: https://github.com/ros2/rosidl_dds.git
@@ -383,7 +383,7 @@ repositories:
   ros2/system_tests:
     type: git
     url: https://github.com/ros2/system_tests.git
-    version: e76c7af724dd47d742a8ff35dc1319a7473b5322
+    version: 36137c53e6e49edf39311513f9d1e53f7edc36dc
   ros2/test_interface_files:
     type: git
     url: https://github.com/ros2/test_interface_files.git
```

my hypothesis is that this error was solved in `https://github.com/ros2/system_tests/pull/580` (but being honest I'm not sure) or another pr I didn't found.

CC: @Crola1702

--- a/ros2param/test/test_verb_load.py
+++ b/ros2param/test/test_verb_load.py
@@ -164,6 +164,7 @@ class TestVerbLoad(unittest.TestCase):
         proc_output,
         rmw_implementation
     ):
+        cls.rmw_implementation = rmw_implementation
         rmw_implementation_filter = launch_testing_ros.tools.basic_output_filter(
             filtered_rmw_implementation=rmw_implementation
         )
@@ -328,6 +329,9 @@ class TestVerbLoad(unittest.TestCase):
             )
 
     def test_verb_load_wildcard(self):
+        # Skip for Connext DDS as license notifications interfere with YAML parsing
+        if self.rmw_implementation == 'rmw_connextdds':
+            raise unittest.SkipTest()
         with tempfile.TemporaryDirectory() as tmpdir:
             # Try param file with only wildcard
             filepath = self._write_param_file(tmpdir, 'params.yaml', INPUT_WILDCARD_PARAMETER_FILE)

--- a/ros2param/test/test_verb_load.py
+++ b/ros2param/test/test_verb_load.py
@@ -329,9 +329,10 @@ class TestVerbLoad(unittest.TestCase):
             )
 
     def test_verb_load_wildcard(self):
-        # Skip for Connext DDS as license notifications interfere with YAML parsing
         if self.rmw_implementation == 'rmw_connextdds':
-            raise unittest.SkipTest()
+            raise unittest.SkipTest(
+                'Skip for Connext DDS as license notifications interfere with YAML parsing'
+            )
         with tempfile.TemporaryDirectory() as tmpdir:
             # Try param file with only wildcard
             filepath = self._write_param_file(tmpdir, 'params.yaml', INPUT_WILDCARD_PARAMETER_FILE)

